doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
+Also note that this is only required for the Network Intensive profile.  Example
+CNI implementations which meet these requirements include Multus and DANM.  For
+more detailed feature comparison of CNI multiplexers/meta-plugins see Table 3-1 below.
+- **CNI Plugin (Additional)**: this is a CNI plugin that is used to provide
+additional networking needs to Pods, that aren't provided by the default CNI plugin.
+This can include connectivity to underlay networks via accelerated hardware devices.
+- **Device Plugin**: this is a Kubernetes extension that allows for the management
+and advertisement of vendor hardware devices. In particular, devices such as
+FPGA, SR-IOV NICs, SmartNICs, etc. can be made available to Pods by using Device Plugins.
+Note that alignment of these devices, CPU topology and Huge Pages will need the use
+of the [Topology Manager](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/).
+- **External / Application Load Balancing**: As Kubernetes Ingress, Egress and Services have no
+support for all the protocols needed in telecommunication environments
+(Diameter, SIP, LDAP, etc) and their capacity is limited, the architecture includes
+the use of alternative load balancers, including external or ones built into the
+application. Management of external load balancers must be possible via Kubernetes API objects.
+- **Other Features**: these additional features that are required by the
+networking solution as a whole, may be delivered by the **"Default CNI Plugin"**,
+or the **"CNI multiplexer/meta-plugin"** if it is deployed. For example,
+the integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+- **Service Mesh**: The well known service meshes are "application service meshes" that address and
+interact with the application layer 7 protocols (eg.: HTTP) only. Therefore,
+their support is not required in this architecture, as these service meshes are
+outside the scope of the infrastructure layer of the CNTT stack.
 
-A CNI metaplugin/CNI multiplexer has the capability to attach several
-interfaces, using different other CNI plugins, to a pod. Note that the different
-network characteristics of the interfaces might require different networking
-technologies, which would potentially require different CNI plugins.
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
 
-To comply with `req.inf.ntw.08`, inter node communication must be served by a
-CNI plugin which complies with the default [Kubernetes networking assumptions](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).
+<!--The above diagram is maintained here: https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
```

Comment:
"CNI Plugin Additional" should connect to a non eth0 interface.

Reply:
It does?  On my version it is connected to either `net0` or `net1`.

Reply:
Is this ok to resolve @CsatariGergely?

Reply:
Correct, the 0-s confused me.